ironfish/src/network/peers/addressManager.ts

```diff
@@ -77,8 +80,11 @@ export class AddressManager {
     const inUsePeerAddresses: PeerAddress[] = peers.flatMap((peer) => {
       if (
         peer.state.type === 'CONNECTED' &&
-        !peer.getConnectionRetry(ConnectionType.WebSocket, ConnectionDirection.Outbound)
-          .willNeverRetryConnecting
+        !this.peerManager.getConnectionRetry(
```

Comment:
nit: A little inconsistent that we pass `PeerManager.peers` into this function but get `PeerManager.peerCandidateMap` by passing in the whole `PeerManager` into the constructor. Maybe we could do both one way or another